releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,16 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+## 1.2.0 - May 24, 2024
+
+### Features
+
+* Added support for loading feature flags from Azure App Configuration.
+
+### Enhancement
+
+* Updated the reterival of configuration settings from provider to be more efficient.
```

Comment:
"more efficient" can be confusing. Is it using less memory?

Improved the performance of configuration loading time.

Reply:
Actually, it would be, don't have numbers. Only noticed the issue when large key value sets were selected, like loading thousands of keys.

But updated to your comment.

Reply:
I thought it's time only based on your original note (_slower to return than expected_). If it's the overall perf, we can tweak the notes (based on what you said):

Improved the performance of loading a large set of key-values.

Reply:
It technically did make loading a single key faster. Just that it's such a small amount that it isn't noticeable as an issue to large key counts.

Also, it was both taking up extra memory and taking longer by how the keys were being returned.

Reply:
Okay. Based on what you said:

Improved the performance of data loading from Azure App Configuration, especially when you have a large set of key-values.